content/pop-recovery.md

```diff
@@ -74,55 +103,43 @@ And now the existing hard drive can be accessed by going to the `/mnt` folder.
 
 ## Chroot
 
-<u>chroot</u> is the way to run commands as if the existing operating system had been booted.  Once these commands are run, then package manager (<u>apt</u>) and other system level commands can be run.
+`chroot` is the way to run commands as if the existing operating system had been booted.  Once these commands are run, then package manager (`apt`) and other system level commands can be run.
 
-The EFI partition is usually around 512MB so that would be the partition that we replace in the next command. The Recovery Partition is around 4GB as well.
+The EFI partition is the next partition to be mounted. To help identify it, this partition is usually around 512MB, and is labelled as `/boot/efi`.
 
 | **SATA Drives**                       | **NVMe Drives**                          |
 |:-------------------------------------:|:----------------------------------------:|
-| sudo mount /dev/sda1 /mnt/boot/efi    | sudo mount /dev/nvme0n1p1 /mnt/boot/efi  |
+| ```sudo mount /dev/sda1 /mnt/boot/efi```    | ```sudo mount /dev/nvme0n1p1 /mnt/boot/efi```  |
 
 ```bash
 for i in /dev /dev/pts /proc /sys /run; do sudo mount -B $i /mnt$i; done
 sudo cp -n /etc/resolv.conf /mnt/etc/
 sudo chroot /mnt
 ```
 
-With this last command you will have root access to your installed system. You can also access your files with <u>files</u> via "Other Locations" > Computer > /mnt.
+With this last command you will have root access to your installed system. Once the drive is accessed, commands for maintenance can be run on the installed system. For example, [package manager repair commands](article/package-manager-pop). You can also access your files with <u>files</u> via "Other Locations" -> "Computer" -> "mnt."
```

Comment:
Missing comma in the interdictory phrase.

`command you` -> `command, you`

Reply:
Fixed.